Makefile

```diff
@@ -166,18 +155,23 @@ install-gitlab-dependency:
 
 check-install:
 	jupyter serverextension list
-	jupyter server extension list
```

Comment:
Is this no longer required/useful? If it is not the command should also be removed from the validation instructions in the README and installation documentation.

Reply:
I agree, but we need to keep `jupyter server extension list` (which comes from [`jupyter_server`](https://github.com/jupyter-server/jupyter_server/blob/62607ce28576e50358217534161f9e7c4aab2611/jupyter_server/extension/serverextension.py#L359)) and remove `jupyter serverextension list` (which comes from [`jupyter notebook`](https://github.com/jupyter/notebook/blob/26626343384195a1f4f5461ba42eb3e133655976/setup.py#L143)).

Reply:
oh did I mix them up? I meant to delete the old redundant one. And I'll double check the docs